monitors/filterWords.js

```diff
@@ -0,0 +1,44 @@
+exports.run = async (client, msg) => {
+  const modlog = msg.guild.settings.modLog;
+  if (msg.deletable) {
```

Comment:
Instead of making a christmas tree, remove some level of indentations with negative check + return.